Makefile

```diff
@@ -97,14 +97,9 @@ testprereqs: $(KUBEBUILDER) $(KUSTOMIZE)
 test: testprereqs fmt lint ## Run tests
 	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./controllers/... ./baremetal/... -coverprofile ./cover.out; cd api; go test -v ./... -coverprofile ./cover.out
 
-.PHONY: test-integration
-test-integration: ## Run integration tests
-	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -tags=integration ./test/integration/...
-
```

Comment:
Just curios why we are removing it? I don't even know if we were using it :)

Reply:
Because we weren't ;). The directory ./test/integration doesn't even exist, the makefile target must have been added spuriously

Reply:
+